docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM alpine:latest AS builder
+
+RUN apk add --no-cache \
+    git \
+    make \
+    cmake \
+    python3 \
+    zlib-dev \
+    curl-dev \
+    clang \
+    boost-dev
+
+WORKDIR /home/cubic-server
+
+COPY CMakeLists.txt .
+COPY cubic-server cubic-server
+COPY generators generators
+COPY blocks.cmake .
+COPY boost.cmake .
+COPY c_flag_overrides.cmake .
+COPY cxx_flag_overrides.cmake .
+
+RUN cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DUSE_CLANG=true -DNO_GUI=true
+
+RUN cmake --build build -j`nproc --ignore=1`
+
+FROM alpine:latest AS runner
+
+# curl-dev is needed because the library is not linked statically
+RUN apk add --no-cache \
+    libstdc++ \
+    curl-dev
```

Comment:
curl-dev might not be needed, curl should be enough (Not sure about that though)

Reply:
It is needed
![image](https://github.com/CubicMC/cubic-server/assets/60519602/d8633d32-d633-4fd5-81bd-0ecb65898b0d)